Resources/Prototypes/_RMC14/Entities/Objects/Devices/command_tablet.yml

```diff
@@ -0,0 +1,22 @@
+- type: entity
+  parent: BaseItem
+  id: RMCTabletCO
+  name: command tablet
+  description: A portable command interface used by top brass, capable of issuing commands over long ranges to their linked computer. Built to withstand a nuclear bomb.
+  components:
+  - type: Sprite
+    sprite: _RMC14/Objects/Devices/command_tablet.rsi
+    state: cotablet
+  - type: Item
+    size: Normal
+  - type: ActivatableUI
+    key: enum.MarineCommunicationsComputerUI.Key
+  - type: ActivatableUIRequiresAccess
+  - type: AccessReader
+    access: [ [ "CMAccessCommand" ] ]
+    key: enum.MarineCommunicationsComputerUI.Key
```

Comment:
The linter fail is because this doesn't need to be here, there is no key property in AccessReader.
It is needed in ActivatableUI and UserInterface though.
```suggestion
```